OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -153,10 +168,11 @@ open class OptimoveConfigBuilder: NSObject {
         super.init()
     }
 
-    @discardableResult public func setCredentials(optimoveCredentials: String?, optimobileCredentials: String?) -> OptimoveConfigBuilder {
```

Comment:
technically this is breaking public contract now :) but assuming nobody should have been using it + it's not documented + only 1-2 clients use delayed init + on android we dont have it + if they did use it would break app and they would spot i think

1. proceed with the change: having 2 ways to set credentials on builder is not lean / confusing / raises q "which one should i use"
```
  let config = OptimoveConfigBuilder(optimoveCredentials: "", optimobileCredentials: "")
            .setCredentials(optimoveCredentials: "", optimobileCredentials: "")
```
2. dont have it as major release to not make android also major / scare all customers from updating